scripts/ghutils.py

```diff
@@ -152,6 +153,20 @@ def merge_pr(repo, pull_request_number, commit, message, retries):
         else:
             raise Exception("There was a failure updating the pull request.")
 
+    pr_repo = repo.github.repository(*pull_request.head.repo)
+    pr_ref = pr_repo.ref("heads/{ref}".format(ref=pull_request.head.ref))
+    try:
+        pr_branch_deleted = pr_ref.delete()
```

Comment:
Looks good.  As a side question, is there a need to confirm if the ref exists before we execute the delete?

Reply:
I guess it depends why you want the information. It's pretty unlikely that it would have been deleted after the merge on 146 and before this attempt but if it is gone, the code won't error so it's not going to cause the build to fail.